armi/tests/refTriso-bp.yaml

```diff
@@ -0,0 +1,145 @@
+# A full blueprint file that contains assems made with ComponentGroups
+blocks:
+    fuel: &block_fuel
+        fuel: &component_fuel_fuel
+            shape: Hexagon
+            material: UZr
+            Tinput: 25.0
+            Thot: 600.0
+            ip: 0.0
+            mult: 1.0
+            op: 10.0
+    fuel2: &block_fuel2
+        triso compact: 
+            shape: Circle
+            blends: 
+              - triso particle
+            blendFracs: 
+              - 0.350
+            material: Graphite
+            Tinput: 25.0
+            Thot: 600.0
+            mult: 20.0 
+            id: 0.0
+            od: 1.2446
+        duct: 
+            shape: Hexagon
+            material: UZr
+            Tinput: 25.0
+            Thot: 600.0
+            ip: 9.0
+            mult: 1.0
+            op: 10.0
+        coolant: 
+            shape: DerivedShape
+            material: Graphite
+            Tinput: 25.0
+            Thot: 600.0
+
+components:
+    # triso dims from Table 4 in TRISO benchmark
+    kernel:
+        shape: Sphere
+        material: UZr
+        Tinput: 25.0
+        Thot: 600.0
+        id: 0.0
+        mult: 1.0
+        od: 0.010625
+    buffer:
+        shape: Sphere
+        material: Graphite
+        Tinput: 25.0
+        Thot: 600.0
+        id: kernel.od
+        mult: 1.0
+        od: 0.015625
+    ipyc:
+        shape: Sphere
+        material: Graphite
+        Tinput: 25.0
+        Thot: 600.0
+        id: buffer.od
+        mult: 1.0
+        od: 0.017375
+    sic:
+        shape: Sphere
+        material: SiC
+        Tinput: 25.0
+        Thot: 600.0
+        id: ipyc.od
+        mult: 1.0
+        od: 0.019125
+    opyc:
+        shape: Sphere
+        material: Graphite
+        Tinput: 25.0
+        Thot: 600.0
+        id: sic.od
+        mult: 1.0
+        od: 0.021125
+
+component groups:
+    triso particle:
+      kernel:
+        mult: 1000
+      buffer:
+        mult: 1000
+      ipyc:
+        mult: 1000
+      sic:
+        mult: 1000
+      opyc:
+        mult: 1000
```

Comment:
Should the multiplicity here be 1 for each component? In that there is one `kernel` in a `triso particle` but maybe thousands of `triso particle` components in the `triso compact` component with the blend.